sdk/storage/azure-storage-blob/azure/storage/blob/_container_client.py

```diff
@@ -1048,37 +1089,29 @@ def delete_blobs(self, *blobs, **kwargs):
         Soft deleted blobs or snapshots are accessible through :func:`list_blobs()` specifying `include=["deleted"]`
         Soft-deleted blobs or snapshots can be restored using :func:`~BlobClient.undelete()`
 
-        :param blobs: The blobs to delete. This can be a single blob, or multiple values can
+        :param blobs:
+            The blobs to delete. This can be a single blob, or multiple values can
             be supplied, where each value is either the name of the blob (str) or BlobProperties.
-        :type blobs: str or ~azure.storage.blob.BlobProperties
+
+            ..note::
+                When then blob type is dict, here's a list of keys you can set:
+                blob name: 'name'
+                container name: 'container'
+                snapshot you want to delete: 'snapshot'
+                whether to delete snapthots when deleting blob: 'delete_snapshots'
+                if the blob modified or not: 'if_modified_since', 'if_unmodified_since'
+                match the etag or not: 'etag', 'match_condition'
+                lease id or lease client: 'lease_id'
```

Comment:
This should be keyword "lease" - we should re-use the parameter names already being used by the "delete_blob" API

Reply:
lease is an attribute for BlobProperties which is a LeaseProperties object which doesn't have lease id, so if user pass lsit(BlobProperties), lease will be taken as a lease, while it's not.